src/input/search-input.tsx

```diff
@@ -23,6 +24,9 @@ const SearchInput: React.FunctionComponent<Props> = (props) => {
         if (!props.disabled) {
             props.onFilterChange('');
         }
+        if (props.onSearchCleared && props.value.length > 0) {
+            props.onSearchCleared();
```

Comment:
Could you  just pass the value to the given function? That it only should be invoked when it's length is > 0 can be handled on the other side.

Reply:
Other than that, it looks fine

Reply:
ok done